packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -264,6 +271,20 @@ class DestinationFromServiceRetriever {
     return { encoded, decoded: decodeJwt(encoded) };
   }
 
+  private static checkDestinationForCustomJwt(destination: Destination): void {
```

Comment:
[q] This is marked as a change here, but I guess nothing changed for the two private static methods?

Reply:
Yes, they were just moved because private has to come before public